examples/experimental/h3-grid/src/h3-utils.js

```diff
@@ -0,0 +1,59 @@
+import {polyfill, getRes0Indexes, h3GetFaces, geoToH3} from 'h3-js';
+
+// Number of hexagons at resolution 10 in tile x:512 y:512 z:10
+// {west: 0, north: 0, east: 0.35156, south: -0.35156}
+const HEX_COUNT_ZOOM_10_RES_10 = 135439;
```

Comment:
Note that this number will change per tile, so if you really wanted to be conservative you'd choose a tile with a pentagon (smallest area, as shown here: https://observablehq.com/@nrabinowitz/h3-area-variation).